Plot.py

```diff
@@ -57,5 +60,8 @@ def createResponseTimeFigure():
   print ("S.E values:")
   print (df.sem())
   
+def filterNegativeNumbers(arr):
+  return [x for x in arr if x >= 0]
```

Comment:
move _filterNegativeNumbers(arr)_ to top of document

Reply:
fixed